erclient/client.py

```diff
@@ -215,7 +215,7 @@ def time_converter(t):
 
         if response and response.ok:
             res_json = response.json()
-            if ('data' in res_json):
+            if ('data' in res_json and len(res_json['data'])):
```

Comment:
I think we can bring this in and just say the following. We can rely on the truthiness of whatever data is at res_json["data"]

```
if "data" in res_json and res_json["data"]:
```
Have we seen a case where res_json was None?

Reply:
I have not personally, but it couldn't hurt to add a check for None on line 216